_drafts/2017-07-27-taxonomy-suite.md

```diff
@@ -311,7 +312,7 @@ filter_taxa(ex_taxmap, startsWith(taxon_names, "t"))
 * Wikidata
 * Wikicommons
 
-The only site of the four above that's only taxonomy is Wikispecies - for the others you could use `wikitaxa` to do any searches, but we look for and parse out taxonomic specific items in the wiki objects that are returned.
+Only Wikispecies is focused on taxonomy - for the others you could use `wikitaxa` to do any searches, but we look for and parse out taxonomic specific items in the wiki objects that are returned.
 
 We released a new version (`v0.1.4`) earlier this year. Big thanks to [Ethan Welty](https://github.com/ezwelty) for help on this package.
 
```

Comment:
Need a conclusion saying how/why the taxonomic suite is greater than the sum of its parts. What are they key compenents still missing? Does rOpenSci plan to maintain a robust suite of taxonomic tools? Any way to be specific about why this is impt to rOpenSci rather than the whole R biologist world?